test/tests.js

```diff
@@ -176,6 +176,21 @@ test('Bad Numbers', function() {
 	});
 });
 
+test('Missing arguments', function() {
+	throws(function(){
+		var x = jsep("check(,)");
+	}, "detects missing argument (all)");
+	throws(function(){
+		var x = jsep("check(,1,2)");
+	}, "detects missing argument (head)");
+	throws(function(){
+		var x = jsep("check(1,,2)");
+	}, "detects missing argument (intervening)");
+	throws(function(){
+		var x = jsep("check(1,2,)");
```

Comment:
@EricSmekens why this check. I think it's supported in JavaScript.  We cannot upgrade due to this change.

Reply:
We might want to add this as a optional configuration parameter you can pass in when you create JSEP to ignore this check.
Does that sound like a good idea?